resources/helm/v1.1/istio/charts/galley/templates/clusterrole.yaml

```diff
@@ -40,9 +40,8 @@ rules:
 - apiGroups: ["extensions"]
   resources: ["ingresses"]
   verbs: ["get", "list", "watch"]
-- apiGroups: ["extensions"]
-  resources: ["deployments/finalizers"]
-  resourceNames: ["istio-galley"]
+- apiGroups: [""]
+  resources: ["namespaces/finalizers"]
   verbs: ["update"]
```

Comment:
Hmmm. What's this? Galley can't touch namespaces, since they are a cluster-scoped resource.

Reply:
You're right; I will file an issue to track this

Reply:
Must have come in with the rebase to 1.4.6

Reply:
Note: I've created https://issues.redhat.com/browse/MAISTRA-1241 for this.